shared/src/store/action-reducers/transfer.ts

```diff
@@ -131,6 +180,26 @@ export namespace TransferActionReducers {
         rights: 'trainer',
     };
 
+    export const deleteTransfer: ActionReducer<DeleteTransferAction> = {
```

Comment:
What is the purpose of this action, and how and when would I use it?

Reply:
These action is there already, this could be used later in the UI to let vehicles arrive immediately. 
Right now only the function declared also in this file is used, to let all elements arrive immediately when a transferPoint gets deleted.

Reply:
Ok. I'm not sure the name is that great, but I'd be fine with it for now.